CHANGELOG.md

```diff
@@ -162,12 +125,53 @@
   ([PR #531](https://github.com/jasmin-lang/jasmin/pull/531);
   fixes [#525](https://github.com/jasmin-lang/jasmin/issues/525)).
 
-- Add more warning options:
-    - `-wduplicatevar`: warns when two variables share the same name;
-    - `-wunusedvar`: warns when a declared variable is not used.
+# Jasmin 2023.06.3 — 2024-04-10
 
-  ([PR #605](https://github.com/jasmin-lang/jasmin/pull/605)).
-  Warning this is a **breaking change**.
+## New features
+
+- The type system for constant time now ensures that division and modulo
+  operators may only be used with public arguments.
+  This ensures that problems like KyberSlash (https://kyberslash.cr.yp.to/) do
+  not occur.
+  ([PR #722](https://github.com/jasmin-lang/jasmin/pull/722)).
```

Comment:
The part about speculative constant time is removed, is this on purpose? Also, the sentence mentioning "KyberSlash" is not in the CHANGELOG.md of release 2023.06.3, not sure if that matters.

Reply:
Yes, it is on purpose: there is no *speculative*-CT checker in 2023.06.

Reply:
Fair enough.